### DISCUSSION TITLE
curl errors making me cry

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#3222] [Questions] [answered]
### Steps to reproduce
1. Go to apps; in this case try to install SSO and SAML Auth
2. Click Download and Enable
3. BOOM

### Expected behavior 
It should have a little spin circle and download without issue
### Actual behavior 
![morecurlissues](https://github.com/nextcloud/all-in-one/assets/78336592/b7a10788-f0b9-4da3-a8cb-54d82b76d145)

### Host OS 
Windows 11 Docker Desktop

#### Nextcloud AIO version 
27.0.2.1
#### Current channel 
Latest
#### Other valuable info 
Enforce IPv4 wherever curl is called i.e. curl -4

**Comment by szaimen** [comment, marked as answer]
I want to clarify that we cannot enforce ipv4 in curl in Nextcloud. So you need to fix your networking instead.

**Reply by FlattusBlastus** [reply]
I fear that its the IPv6 implementation in Windows Docker Desktop.
Stuck in Windows because of the new machine doesnt have full driver support in Linux.
but..... https://ryzenai.docs.amd.com/en/latest/inst.html
I get decent AI response with my 7940hs processor.
